presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt

```diff
@@ -15,16 +15,187 @@
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.android254.presentation.R
+import com.android254.presentation.common.components.DroidconAppBarWithFeedbackButton
+import com.android254.presentation.common.components.OrganizedBySection
 import com.android254.presentation.common.theme.DroidconKE2022Theme
+import com.android254.presentation.models.OrganizingTeamMember
+import com.google.accompanist.flowlayout.FlowRow
+import com.google.accompanist.flowlayout.MainAxisAlignment
+import com.google.accompanist.flowlayout.SizeMode
 
 @Composable
-fun AboutScreen() {
-    Text(modifier = Modifier.testTag("heading"), text = "About Screen")
+fun AboutScreen(
+    aboutViewModel: AboutViewModel = hiltViewModel(),
+) {
+    val teamMembers: List<OrganizingTeamMember> = aboutViewModel.organizingTeamMembers.value
+    val stakeHolderLogos = aboutViewModel.stakeHolderLogos.value
+    val sampleImageUrl = aboutViewModel.sampleImageUrl
+
+    Scaffold(
+        topBar = {
+            DroidconAppBarWithFeedbackButton(
+                onButtonClick = {
+                    // TODO navigate to feedbackScreen
+                },
+                userProfile = sampleImageUrl
+            )
+        }
+    ) { paddingValues ->
+
+        Column(
+            Modifier
+                .padding(paddingValues)
+                .verticalScroll(rememberScrollState())
+                .testTag("about_screen")
+        ) {
+            AboutDroidConSection(
+                droidconDesc = stringResource(id = R.string.about_droidcon),
+                droidconImage = sampleImageUrl,
+            )
+
+            Spacer(modifier = Modifier.height(40.dp))
+
+            OrganizingTeamSection(
+                modifier = Modifier,
+                organizingTeam = teamMembers,
+                onClickMember = {
+                    // TODO navigate to team screen
+                }
+            )
+
+            Spacer(modifier = Modifier.height(40.dp))
+
+            OrganizedBySection(
+                modifier = Modifier.padding(start = 20.dp, end = 20.dp),
+                organizationLogos = stakeHolderLogos
+            )
+
+            Spacer(modifier = Modifier.height(40.dp))
+        }
+    }
+}
+
+@Composable
+fun AboutDroidConSection(
```

Comment:
`AboutDroidConSection` -> `AboutDroidconSection`

Reply:
@chepsi I have updated this